pages/treeherder.py

```diff
@@ -15,7 +15,11 @@
 
 class TreeherderPage(Base):
 
+    _first_resultset_datestamp_locator = (By.CSS_SELECTOR, '.result-set .result-set-title-left > span a')
+    _pin_all_jobs_locator = (By.CSS_SELECTOR, '.result-set .glyphicon-pushpin')
```

Comment:
This should be within `ResultSet` region, as there's a pin all icon for each result set